packages/devtools_app/lib/src/memory/memory_allocation_table_view.dart

```diff
@@ -92,6 +96,72 @@ class AllocationTableViewState extends State<AllocationTableView>
       final Tracker item = trackerData.selectionNotifier.value.node;
       if (item is TrackerMore) trackerData.expandCallStack(item);
     });
+
+    addAutoDisposeListener(controller.selectTheSearchNotifier, () {
+      if (_trySelectItem()) {
+        setState(() {
+          controller.closeAutoCompleteOverlay();
+        });
+      }
+    });
+
+    addAutoDisposeListener(controller.searchNotifier, () {
+      if (_trySelectItem()) {
+        setState(() {
+          controller.closeAutoCompleteOverlay();
+        });
+      }
```

Comment:
put this into a helper method instead of duplicating with listener above

Reply:
Done.